### ISSUE TITLE
It seems the caching of multiple-to-hundreds of GB on the users browser should be top priority bug fix

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#8488]
**Describe the bug**
My site traffic is down by 75%. Every page load the user is asked to store between 40GB-200GB of data.

I don't blame them for leaving. Thanks to the troopers who stuck it out lol.

**Comment by para2022** [comment]
I have same problem and my traffic dropped off completely over the past few months. To many bugs for users to handle. I got drained with all the moaning and complaints then users started deleting their channels for no reason. But its cos they couldn't maintain a channel because if it wasn't one thing it was another and id run out of excuses. And yes if it was me i would have left long ago too. Id just go elsewhere

**Comment by walt93** [comment]
Same. I'm researching another solution. PlayTube looks decent. I'm going to fuck around with writing a script to export the database to PlayTube. Maybe try using PlayTube a a front end and AVideo for a CMS for a while.

**Comment by para2022** [comment]
I did the reverse by writing scripts for playtube >> Avideo. Lol

**Comment by para2022** [comment]
If i was in Dans shoes. I would sit down at the drawing board and rethink the cache and the search as primary todos. Maybe set this as a goal for next update release for the end of the year if not sooner and meanwhile focusing on the small bug reports but if they are not critical bug fixes maybe address those as a lower priority whilst spending more time and focus on improving search/cache. I just think this absolutely has to happen. Too many will jump ship to other platforms. He works hard but this is unacceptable having critical functions of the platform not working properly. Its a shame cos the platform is beautiful but is wrecked without a proper working search function and a cache that isnt affecting realtime functions like, edit, view updates, likes ect. Playtubes search was practically instant on a huge site. Dan needs to take inspiration from those search function?

**Comment by walt93** [comment]
Ok I think we got a lot closer to the root cause here @para2022 - please follow along on this thread https://github.com/WWBN/AVideo/issues/8500

TL;dr the script creating CachesInDB was breaking, causing there to be no index for that table. This caused massive slowdowns and _appears (so far)_ to have been the cause of the browser asking to store massive amounts of data on the users device.
